### ISSUE TITLE
Icon more in style with gnome symbolic icons

### ISSUE THREAD

**Issue by corebots** [Rmano/gse-touchpad-onoff#1]
Hello and thank you for the extension! I've replaced your provided icons with two symbolic icons more in style with gnome symbolic icons. Would you be interested in a contribution? Of course, you might have reasons for using your icons and I respect that. But I thought I'd ask.... It's the one on the left in the screenshots. I'm also attachind them to this post in case you want them. Feel free to close otherwise.
![image](https://github.com/Rmano/gse-touchpad-onoff/assets/12252430/fe63a686-c5a6-4c18-b5f7-d557e900acd2)
![image](https://github.com/Rmano/gse-touchpad-onoff/assets/12252430/2bfb6fb4-9181-49a4-a512-98aee19d1142)
![touchpadoff](https://github.com/Rmano/gse-touchpad-onoff/assets/12252430/17ed0ae9-1da6-4c62-ba5b-b605ce637beb)
![touchpadon](https://github.com/Rmano/gse-touchpad-onoff/assets/12252430/c8c48fe8-2d86-4bfa-9233-d801b80fb645)

**Comment by Rmano** [comment]
Hm! I like them! They are clearly more in style with the others, although I personally prefer the touch of color... but I understand this is clearly better from a graphical point of view.

But yes, I'll add them. Would you like to make a PR? Do you prefer I do the thing (will obviously attribute it correctly 😉)

(Can also add your icons and modify them with the red/green thing, and add an option defaulting to yours...)

Thnaks!

**Comment by Rmano** [comment]
BTW, probably my lack of knowledge --- opening the icons in Inkscape, they are basically invisible (they have no stroke paint defined). I suppose that is normal and made to avoid overriding the user's color scheme, but... ¿could you give a hint on how to edit/play with them? Or a link on how to draw icons for Gnome?

**Comment by Rmano** [comment]
Never mind, I did it ;-).

**Comment by Rmano** [comment]
@corebots , would you like  to test it? Thanks!

**Comment by corebots** [comment]
Hello! Sorry for the late response, I'm traveling Asia currently and not opening my laptop every day. 

First of all thanks for accepting the icons :) I've updated your extension just now and it seems to work nicely! 

Secondly, I have never contributed like this to an extension (or other project on github) so even if I'd to do it via a PR, I'll first need to find out how to do that (for possible future contributions, since now you've done all the work).

Then to your question about drawing the icons - what I do is to make a screenshot of an existing symbolic icon in the panel, then put it in Figma (or well Inkscape) make a frame of 16x16 px and draw the shapes with rounded corners so that it visually aligns with the screenshot I took as reference, leaving a gap of free space around them. In this case, I took a square of 12x12 px as a base + the stroke. Not really a proper guide I know, but it works :)
